### PR TITLE
Added some Android devices

### PR DESCRIPTION
**Pull request by GregOriol**

### CODE REVIEW

File: regexes/device/mobiles.yml [hunk 1]
```diff
@@ -5100,19 +5139,19 @@ Samsung:
...
-    - regex: '(?:SAMSUNG-)?SM-G7105'
+    - regex: '(?:SAMSUNG-)?SM-G710[25]'
       model: 'GALAXY Grand 2'
       device: 'phablet'
     - regex: '(?:SAMSUNG-)?SM-G53(?:0F|0FZ|0Y|0H|0FZ|1F|1H)'
       model: 'GALAXY Grand Prime'
       device: 'phablet'
     - regex: '(?:SAMSUNG-)?SM-G532F'
-      model: 'GALAXY Grand Prime Plus'
+      model: 'GALAXY J2 Prime/Grand Prime Plus'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: We should decide to use only one name here. Maybe the more familiar / common name if possible.

File: regexes/device/mobiles.yml [hunk 2]
```diff
@@ -5066,10 +5099,16 @@ Samsung:
       model: 'GALAXY J7+ (C7)'
     - regex: '(?:SAMSUNG-)?SM-G610[FMY]'
       model: 'GALAXY J7 Prime'
+    - regex: '(?:SAMSUNG-)?SM-G615[F]'
+      model: 'GALAXY J7 Max'
     - regex: '(?:SAMSUNG-)?SM-J730(?:F|G|GM)'
       model: 'GALAXY J7 Pro'
+    - regex: '(?:SAMSUNG-)?SM-J701(?:F|M)'
+      model: 'GALAXY J7 Nxt/Neo/Core'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: would use one of the names only here as well

File: regexes/device/mobiles.yml (hunk 1 above)
Comment (GregOriol) [reply regexes/device/mobiles.yml]: I don't disagree, but I could not decide as both "Grand Prime" and "J2" exist as other variants, and I don't know if one of these two is more popular.

File: regexes/device/mobiles.yml (hunk 2 above)
Comment (GregOriol) [reply regexes/device/mobiles.yml]: Same as the other comment: I don't disagree, but don't know which one could be best...
The names for this one seem to be different by region:
* Nxt in India
* Neo in LATAM
* Core in Philippines

**Comment by GregOriol** [comment]
@sgiehl So, what would you suggest regarding your comments?

**Comment by sgiehl** [comment]
@GregOriol  sorry for the delay. To be honest I'm not sure how to handle that best. But using multiple names might be a bit long. Maybe we could use the model name having most google results?

**Comment by GregOriol** [comment]
Names simplified using Google results, I left the alternatives as comments in the yaml, so we can remember/find them later if trends change.

File: regexes/device/mobiles.yml [hunk 3]
```diff
@@ -3407,6 +3417,18 @@ LG:
...
       model: 'V20'
+    - regex: '(?:LG-)?F800(?:S|K|L)?'
+      model: 'V20'
+    - regex: '(?:LG-)?H91[058]'
+      model: 'V20'
+    - regex: '(?:LG-)?H990(?:DS|N|T)?'
+      model: 'V20'
+    - regex: '(?:LG-)?LS997'
+      model: 'V20'
+    - regex: '(?:LG-)?VS995'
```
Comment (sgiehl) [inline regexes/device/mobiles.yml]: this one seems to be a duplication.

Comment (GregOriol) [reply regexes/device/mobiles.yml]: Do you prefer to regroup them into a single regex?

Comment (sgiehl) [reply regexes/device/mobiles.yml]: that's not necessarily needed. but this specific regex is already done some lines before

**Pull request merged**